install-dev/init.php

```diff
@@ -76,13 +76,8 @@
 if (file_exists(_PS_CORE_DIR_.'/app/config/parameters.php')) {
     require_once _PS_CORE_DIR_.'/config/bootstrap.php';
 
-    if (defined('_PS_IN_TEST_') && _PS_IN_TEST_) {
-        $env = 'test';
-    } else {
-        $env = _PS_MODE_DEV_ ? 'dev' : 'prod';
-    }
     global $kernel;
-    $kernel = new AppKernel($env, _PS_MODE_DEV_);
+    $kernel = new AppKernel(_PS_ENV_, _PS_MODE_DEV_);
```

Comment:
can we make it overridable using env variables?

Like we usually do in Symfony https://github.com/mickaelandrieu/resiliency-demo/blob/master/public/index.php#L23

Reply:
Should be done in a new pull request because if we want to do it for the env mode, we should do it for debug and others